Example/Tests/NetworkTests/ManagementApiTests.swift

```diff
@@ -236,6 +236,61 @@ class ManagementApiTests: NetworkBaseTest {
         XCTAssertEqual(result?.publicIds?.first ?? "", uploadedPublicId)
     }
     
+    func testTagsAsArray() {
+        let tagsArray = ["tag1","tag2","tag3"]
+        
+        var expectation = self.expectation(description: "Adding a tag should succeed")
+        
+        var result: CLDTagResult?
+        var error: Error?
+        
+        var uploadedPublicId: String = ""
+        // first upload
+        uploadFile().response({ (uploadResult, uploadError) in
+            if let pubId = uploadResult?.publicId {
+                uploadedPublicId = pubId
+                
+                // test adding a tags
+                self.cloudinary!.createManagementApi().addTag(tagsArray, publicIds: [uploadedPublicId]).response({ (resultRes, errorRes) in
```

Comment:
Where is the check that the actual tags were added to the asset?

Reply:
The response from the addTag call return only an array of affected public ids.
There's no Admin API in iOS which means we cant get the list of tags